src/components/Checkout/BraintreeCheckout.vue

```diff
@@ -448,9 +449,10 @@ export default {
 			this.apollo.mutate({
 				mutation: braintreeDepositAndCheckout,
 				variables: {
+					basketId: cookieStore.get('kvbskt'),
```

Comment:
I'm still concerned about adding in the basket id here when it should already be being added by the BasketLink, but theoretically this shouldn't cause problems.